### ISSUE TITLE
Running chksetup before setup doesn't return OK

### ISSUE THREAD

**Issue by bedroge** [cvmfs-contrib/cvmfs-tutorial-2021#32]
```
[bobtest@ip-0A000008 ~]$ sudo cvmfs_config chksetup
Warning: CernVM-FS map is not referenced from autofs maps (/etc/auto.master)
Warning: autofs service is not running
[bobtest@ip-0A000008 ~]$ sudo cvmfs_config setup
[bobtest@ip-0A000008 ~]$ sudo cvmfs_config chksetup
OK
```

So, we should probably reverse the order of these commands in the client section.

**Comment by rptaylor** [comment]
Yes, a client installation needs either the setup command or the cvmfs-auto-setup  package.

**Comment by boegel** [comment]
@bedroge This can be closed, thanks to #40 ?

**Comment by bedroge** [comment]
Ah, so here it is. I thought we had already changed this when we were running into this issue again today, but apparently there was just an open issue... but it can indeed be closed now.
